.azure-pipelines/ci-scan.yml

```diff
@@ -0,0 +1,25 @@
+pool:
```

Comment:
how will this yml get consumed? I don't see it being consumed in azure-pipelines.yml?

Reply:
There will be separate ci for this
https://dev.azure.com/mseng/AzureDevOps/_build?definitionId=10036&_a=summary